frontend/src/App.vue

```diff
@@ -51,7 +51,32 @@
           cols="3"
           class="d-flex align-center justify-end"
         >
+          <v-menu
+            offset-y
+            transition="slide-y-transition"
+          >
+            <template #activator="{ on }">
+              <v-btn
+                small
+                text
+                fab
+              >
+                <v-icon
+                  class="white--text"
+                  v-on="on"
+                >
+                  mdi-help-circle
+                </v-icon>
+              </v-btn>
+            </template>
+            <v-list class="text-center">
+              <v-list-item @click="showVersionInfo = true">
+                <v-list-item-title>Versionsinfo</v-list-item-title>
```

Comment:
Finde das Andere klingt zu umgangsprachlich
```suggestion
                <v-list-item-title>Versionsinformationen</v-list-item-title>
```